frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -65,12 +69,26 @@ const Map: React.FC<MapProps> = props => {
   useEffect(() => {
     handleViewportChanged();
   }, [agencyId, propertyClassificationId]);
+
+  // fetch GIS base layers configuration from /public folder
+  useEffect(() => {
+    axios.get('basemaps.json').then(result => {
+      setBaseLayers(result.data?.basemaps);
+      setActiveBasemap(result.data?.basemaps?.[0]);
+    });
+  }, []);
+
```

Comment:
I made the choice here to load the **basemaps.json** dynamically via a fetch request (with axios) so we have the ability to override the json configuration upon deployment to Openshift via config maps (like we have done with keycloack.json). 

I believe React/webpack support loading json files via `import data from basemaps.json` but that would mean the json file is included in the JS bundle and we wouldn't be able to override/change it without a code change and re-deploy.